### PR TITLE
INTYGFV-14704: Updated release date for texts.

### PR DESCRIPTION
**Pull request by IamFriberg**

### CODE REVIEW

File: src/main/resources/texts/texterMU_LUAE_NA_v1.2.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<texter xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" giltigFrom="2022-07-05" typ="LUAE_NA" version="1.2" xsi:schemaLocation="http://schema.forsakringskassan.se/vardgivare_sjukvard/texterMU_v1.0.xsd" xmlns="http://schema.forsakringskassan.se/vardgivare_sjukvard">
+<texter xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" giltigFrom="2022-09-01" typ="LUAE_NA" version="1.2" xsi:schemaLocation="http://schema.forsakringskassan.se/vardgivare_sjukvard/texterMU_v1.0.xsd" xmlns="http://schema.forsakringskassan.se/vardgivare_sjukvard">
```
Comment (mww-sopra-steria) [inline src/main/resources/texts/texterMU_LUAE_NA_v1.2.xml:2]: Should be 2022-09-27.

File: src/main/resources/texts/texterMU_LUSE_v1.3.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<texter xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" giltigFrom="2022-07-05" typ="LUSE" version="1.3" xsi:schemaLocation="http://schema.forsakringskassan.se/vardgivare_sjukvard/texterMU_v1.0.xsd" xmlns="http://schema.forsakringskassan.se/vardgivare_sjukvard">
+<texter xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" giltigFrom="2022-09-01" typ="LUSE" version="1.3" xsi:schemaLocation="http://schema.forsakringskassan.se/vardgivare_sjukvard/texterMU_v1.0.xsd" xmlns="http://schema.forsakringskassan.se/vardgivare_sjukvard">
```
Comment (mww-sopra-steria) [inline src/main/resources/texts/texterMU_LUSE_v1.3.xml:2]: Should be 2022-09-27.

**Review by mww-sopra-steria — CHANGES_REQUESTED** [review]

File: src/main/resources/texts/texterMU_LUAE_NA_v1.2.xml (hunk 1 above)
Comment (IamFriberg) [reply src/main/resources/texts/texterMU_LUAE_NA_v1.2.xml]: Fixed

File: src/main/resources/texts/texterMU_LUSE_v1.3.xml (hunk 2 above)
Comment (IamFriberg) [reply src/main/resources/texts/texterMU_LUSE_v1.3.xml]: Fixed

**Review by mww-sopra-steria — APPROVED** [review]

**Pull request merged**